packages/opentelemetry/src/index.ts

```diff
@@ -209,11 +211,16 @@ export async function init(config: OpenTelemetryConfig) {
   tracerProvider.register();
 
   instrumentations.forEach((i) => i.setTracerProvider(tracerProvider));
+}
 
-  // When the process starts shutting down, terminate OTEL stuff.
-  process.on('SIGTERM', () => {
-    tracerProvider.shutdown().catch((error) => console.error('Error terminating tracing', error));
-  });
```

Comment:
Nice! I agree that we don't want to install signal handlers from inside library code. Shifting this responsibility to the caller is the right thing to do.